docs/dev/release-process.md

```diff
@@ -28,5 +28,27 @@ While the `[do not deploy]` can be anywhere in the title, it is recommended to p
 The deployment pipeline will analyse the message of the merge commit (which will be contain the PR title) and make a decision based on it.
 
 ## Releasing to production
-The process to promote a build from `stage` to `production` is currently manually initiated by SRE.
-[This ticket](https://mozilla-hub.atlassian.net/browse/CONSVC-1566) (requires LDAP) deals with automating the process.
+Developers with write access to the Merino repository can initiate a deployment to production after a Pull-Request on the Merino GitHub repository is merged to the `main` branch. This can be done by:
+
+1. opening the [circleci dashboard][circleci_dashboard];
+2. looking up the pipeline named `merino <PR NUMBER>` running in the `main-workflow`; this pipeline should either be in a running status (if the required test jobs are still running) or in the "on hold" status, with the `unhold-to-deploy-to-prod` being held;
+3. once in the "on hold" status, with all the other jobs successfully completed, clicking on the "thumbs up" action on the `unhold-to-deploy-to-prod` job row will approve it and trigger the deployment, unblocking the `deploy-to-prod` job;
+4. developers **must** monitor the [Merino Application & Infrastructure][merino_app_info] dashboard for any anomaly, for example significant changes in HTTP response codes, increase in latency, cpu/memory usage (most things under the infrastructure heading).
+
+[circleci_dashboard]: https://app.circleci.com/pipelines/github/mozilla-services/merino?branch=main&filter=all
+[merino_app_info]: https://earthangel-b40313e5.influxcloud.net/d/Cm83vS57z/merino-application-and-infrastructure?orgId=1&refresh=1m
+
+## What to do if production is broken?
+Don't panic and follow the instructions below:
+
+- depending on the severity of the problem, decide if this warrants [kicking off an incident][incident_docs];
```

Comment:
Yes, I believe this part should be changed / extended by somebody else rather than me. I meant to add this as an initial point of discussion, in order to have _something_. What I wrote is mostly high level and always relevant, regardless of the specifics of the incident and it boils down to this:

- if the problem is trivial and the fix is easy, create a PR, fix the thing and deploy;
- if the problem warrants a more thorough investigation, revert/turn off (if only we had feature flags) the latest change and investigate.

I'd recommend either leaving this text as it is or extending it in a separate PR as you see fit. I believe there's still value in slightly documenting the process to act on things when prod breaks.

Reply:
I may be helpful to indicate that this section is subject to change.

Reply:
I'd rather not mark content as subject to change for now. I believe the guidelines would be valid even if the context around them would change, but I'd defer this to future PRs to settle. (or really leave it to @ncloudioj )

Reply:
> I'd recommend either leaving this text as it is or extending it in a separate PR as you see fit. I believe there's still value in slightly documenting the process to act on things when prod breaks.

Sounds good to me. I think we will further expand this section with detailed guidance in a follow-up.